csharp.html.markdown

```diff
@@ -236,7 +236,8 @@ on a new line! ""Wow!"", the masses cried";
             // Ternary operators
             // A simple if/else can be written as follows
             // <condition> ? <true> : <false>
-            string isTrue = (true) ? "True" : "False";
+            int toCompare = 17;
```

Comment:
@vendethiel  are you suggesting something in the line of `comparisonResult` instead of `isTrue`?